models.rdf

```diff
@@ -66,6 +66,14 @@
       <rdfs:isDefinedBy rdf:resource="http://pcdm.org/models#"/>
     </rdfs:Class>
 
+    <rdfs:Class rdf:about="http://pcdm.org/models#FileSet">
```

Comment:
The case I keep coming back to is a FileSet that represents an act of digitization, and is used to group the files that derive from that digitization event.  I would like to record the technician or vendor who did the digitizing, and the date it was done as metadata on the FileSet. Would that stiill be considered technical metadata or could one use something like dc:creator?